bcda/bcdacli/cli.go

```diff
@@ -658,3 +680,71 @@ func setBlacklistState(cmsID string, blacklistState bool) error {
 
 	return db.Model(&aco).Update("blacklisted", blacklistState).Error
 }
+
+// CCLF file name pattern and regex
+const cclfPattern = `((?:T|P)\.BCD\..*\.ZC\d*)Y(\d{2}\.D\d{6}\.T\d{7})`
+
+var cclfregex = regexp.MustCompile(cclfPattern)
+
+func RenameCCLF(name string) string {
```

Comment:
Do you see value in having this available outside of the cli package? If not, I think we should not export it and make it `renameCCLF`.